### PR TITLE
Fix iMessage extension

### PR DESCRIPTION
**Pull request by yonaskolb**
Setting Presets
- sets the correct app icon
- sets runtime search paths to `$(inherited) @executable_path/Frameworks @executable_path/../../Frameworks`, allow it to lookup frameworks in the application

Fixes https://github.com/pepibumur/xcode-modular-apps-workshop

### CODE REVIEW

File: SettingPresets/Products/app-extension.messages.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ASSETCATALOG_COMPILER_APPICON_NAME: iMessage App Icon
+LD_RUNPATH_SEARCH_PATHS: "$(inherited) @executable_path/Frameworks @executable_path/../../Frameworks"
```
Comment (toshi0383) [inline SettingPresets/Products/app-extension.messages.yml]: > @executable_path/../../Frameworks

Wow, this is new👀

Comment (yonaskolb) [reply SettingPresets/Products/app-extension.messages.yml]: What’s new?

**Comment by yonaskolb** [comment]
@keith

File: SettingPresets/Products/app-extension.messages.yml (hunk 1 above)
Comment (toshi0383) [reply SettingPresets/Products/app-extension.messages.yml]: I mean the location path was new to me.  NVM👋

**Pull request merged**